style.css

```diff
@@ -489,6 +436,96 @@ li {
   border: solid 1px #5e6c84;
 }
 
+.imgSrc {
+  width: 20px;
+  height: 20px;
+}
+
+.work-section-popup.close-popup {
+  display: none;
+}
+
+.work-wrapper {
+  background: rgb(193, 199, 208, 0.7);
+  backdrop-filter: blur(8px);
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+}
+
+.work-card-pop {
+  background-color: rgb(255, 255, 255);
+  width: 90%;
+  margin: auto auto;
+  border-radius: 1.2rem;
+  min-height: 500px;
+  min-height: 70%;
+  height: 97%;
+}
+
+.work-card-pop-close {
+  font-size: 30px;
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+  padding: 10% 5% 0 3%;
+  position: relative;
+}
+
+.work-details-pop {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: flex-start;
+  margin: 0 auto;
+  width: 95%;
+  height: 87%;
+}
+
+.pop-large {
```

Comment:
- [ ] [OPTIONAL] The close button is not aligned properly with the project title, kindly fix this so it would be in the same line
- [ ] [OPTIONAL] The popup image in the desktop view is stretching kindly make the image display without stretching out, this will make the popup looks more organized

![Screenshot_20220722_053227](https://user-images.githubusercontent.com/25227022/180363471-78a60ed2-9dc0-42d1-9210-26ccd4d3c144.png)
![Screenshot_20220722_053247](https://user-images.githubusercontent.com/25227022/180363526-3bc18a2e-1b50-4690-a4c9-7b54f72fa16c.png)